.github/workflows/merge-bot.yml

```diff
@@ -0,0 +1,36 @@
+name: Merge Bot
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - ready_for_review
+      - review_request_removed
+      - review_requested
+      - synchronize
+      - unlabeled
+      - edited
+  pull_request_review:
+    types:
+      - dismissed
+      - submitted
+
+jobs:
+  merge:
+    name: Merge
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'ready') }}
```

Comment:
Will this react if a pull request is good to go (approved, test passing etc.) and _then_ I add ready?

Reply:
yeah

Reply:
All the bot does is tell github "merge when Github thinks its ready"